### ISSUE TITLE
OF 1.3 Compliance: Missing class ofp_meter_multipart_request

### ISSUE THREAD

**Issue by abaruchi** [kytos/python-openflow#274]
Class `ofp_meter_multipart_request` is missing.
Page 73 of 1.3 Spec.
```
struct ofp_meter_multipart_request {
    uint32_t meter_id;       /* Meter instance, or OFPM_ALL. */
    uint8_t pad[4];          /* Align to 64 bits. */
};
OFP_ASSERT(sizeof(struct ofp_meter_multipart_request) == 8);
```

**Comment by renanrodrigo** [comment]
Isn't this a duplicate from #272 ?

**Comment by cemsbr** [comment]
Only the page is different here. @abaruchi, could you take a look?

**Comment by abaruchi** [comment]
@cemsbr You are correct. I doubled checked the spec.
This issue can be closed since it is duplicated (correct one is #272)
